src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -116,6 +117,7 @@ public class LogManagerService extends PluginService {
     Map<String, ComponentLogConfiguration> componentLogConfigurations = new ConcurrentHashMap<>();
     @Getter
     private final CloudWatchLogsUploader uploader;
+    @Getter
```

Comment:
Why do we need a `Getter` on this one now?

Reply:
Some tests needs to monitor provessor`s state.

Reply:
Take my words back, some tests used to need it, but not anymore. Will remove it.